library/core/src/intrinsics.rs

```diff
@@ -1787,6 +1787,8 @@ extern "rust-intrinsic" {
     /// so this rounds half-way cases to the number with an even least significant digit.
     ///
     /// May raise an inexact floating-point exception if the argument is not an integer.
+    /// However, Rust assumes floating-point exceptions cannot be bserved, so this is not something that
```

Comment:
I assume this is a typo? (on both additions)
```suggestion
    /// However, Rust assumes floating-point exceptions cannot be observed, so this is not something that
```

Reply:
*oops* fixed